platform/bootstrap/config.aws-aws.tpl

```diff
@@ -117,9 +46,9 @@
                 "Plugin": "flavor-swarm/worker",
                 "Properties": {
                   "InitScriptTemplateURL": "{{ var "/script/baseurl" }}/worker-init.tpl",
-                  "SwarmJoinIP": "{{ var "/m1/ip" }}",
+                  "SwarmJoinIP": "{{ var "/bootstrap/ip" }}",
                   "Docker" : {
-                    "Host" : "tcp://{{ var "/m1/ip" }}:{{ var "/docker/remoteapi/port" }}"
+                    "Host" : "tcp://{{ var "/bootstrap/ip" }}:{{ var "/docker/remoteapi/port" }}"
```

Comment:
the Docker Host parameter should be localhost (it's for the InfraKit client, and it's always running on the same node as the swarm leader)